lib/rpmsg/rpmsg_virtio.c

```diff
@@ -851,8 +853,8 @@ int rpmsg_init_vdev_with_config(struct rpmsg_virtio_device *rvdev,
 		rpmsg_virtio_wait_remote_ready(rvdev);
 	}
 #endif /*!VIRTIO_DRIVER_ONLY*/
-	vdev->features = rpmsg_virtio_get_features(rvdev);
-	rdev->support_ns = !!(vdev->features & (1 << VIRTIO_RPMSG_F_NS));
+	virtio_get_features(rvdev->vdev, &features);
```

Comment:
return to test

Reply:
As above, this was already an issue, the return status was never checked. But in this case since we will probably keep this function I'll fix it here as part of this series.